### DISCUSSION TITLE
remark-math and remark-mathjax not working with nuxt v2.15.7 and @nuxt/content v1.14.0

### DISCUSSION THREAD

**Discussion by TannerGilbert** [remarkjs/remark#775] [Q&A] [answered]
I created a [Nuxt content blog starter repository](https://github.com/TannerGilbert/Nuxt-Content-Blog-Starter) about a year ago and I used [remark-math](https://github.com/remarkjs/remark-math) and [remark-mathjax](https://github.com/remarkjs/remark-math/tree/main/packages/rehype-mathjax) to render math equations. This works fine with the following version:
* "@nuxt/content": "^1.9.0",
* "nuxt": "^2.14.6",
* "rehype-mathjax": "^3.0.0",
* "remark-math": "^3.0.1",

Now I wanted to update `@nuxt/content` and `nuxt` to their newest versions, but after the update, the equations aren't displayed correctly anymore. 

![math](https://user-images.githubusercontent.com/36239763/125193625-abf12980-e24d-11eb-8e9a-6b71714c727b.png)

Does anyone know what could cause the issue? Any help is greatly appreciated.

**Comment by ChristianMurphy** [comment, marked as answer]
It looks like `@nuxt/content ` uses `remark-parse` 9 (part of remark 13) https://github.com/nuxt/content/blob/0b6a7be6f6287cde880b0d37c88e6461efa167ff/packages/content/package.json#L47
Which means `remark-math` needs to be version >=4 https://github.com/remarkjs/remark-math/releases/tag/4.0.0

**Reply by TannerGilbert** [reply]
@ChristianMurphy Just tried v4.0.0 and it seems to work. Thanks a lot.
